pyscriptjs/tests/integration/test_py_runtime_config.py

```diff
@@ -55,10 +55,13 @@ def test_runtime_config(self, tar_location):
         """,
             extra_head="""
             <py-config>
-                runtimes:
-                - src: "/pyodide/pyodide.js"
-                  name: pyodide-0.20.0
-                  lang: python
+                {
+                    "runtimes": [{
+                        "src": "/pyodide/pyodide.js",
+                        "name": "pyodide-0.20.0",
```

Comment:
I thought we are now using pyodide-0.21.0?

Reply:
This is a test -- that checks if you can load a different runtime locally, i.e. which is not hosted. A sanity check for this test is to also load a different version so that we are sure to load the correct one. You can expand this file and read the comment I have written above. The snippet here only uses the new format instead of the YAML one, hence the change.

CC: @antocuni